<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralWidgetGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="waybillTab">
       <attribute name="icon">
        <iconset>
         <normaloff>Icon/Glyph/Notes2.png</normaloff>Icon/Glyph/Notes2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Waybill</string>
       </attribute>
       <layout class="QGridLayout" name="waybillTabGridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <property name="spacing">
         <number>8</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="requisitionTab">
       <attribute name="icon">
        <iconset>
         <normaloff>Icon/Glyph/Notes.png</normaloff>Icon/Glyph/Notes.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Requisition</string>
       </attribute>
       <layout class="QGridLayout" name="requisitionTabGridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <property name="spacing">
         <number>8</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="driverTab">
       <attribute name="icon">
        <iconset>
         <normaloff>Icon/Glyph/OldMan.png</normaloff>Icon/Glyph/OldMan.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Driver</string>
       </attribute>
       <layout class="QGridLayout" name="driverTabGridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <property name="spacing">
         <number>8</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="vehicleTab">
       <attribute name="icon">
        <iconset>
         <normaloff>Icon/Glyph/Cars.png</normaloff>Icon/Glyph/Cars.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Vehicle</string>
       </attribute>
       <layout class="QGridLayout" name="vehicleTabGridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <property name="spacing">
         <number>8</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="providerTab">
       <attribute name="icon">
        <iconset>
         <normaloff>Icon/Glyph/Provider.png</normaloff>Icon/Glyph/Provider.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Provider</string>
       </attribute>
       <layout class="QGridLayout" name="providerTabGridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <property name="spacing">
         <number>8</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuConnection"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
